viadot/flows/genesys_to_adls.py

```diff
@@ -155,106 +131,3 @@ def gen_flow(self) -> Flow:
 
         add_timestamp.set_upstream(file_names, flow=self)
         adls_bulk_upload.set_upstream(add_timestamp, flow=self)
-
-
-class GenesysReportToADLS(Flow):
```

Comment:
Yes, I am.
It was the first version of Genesys connection, but it was obsolete quickly. We never made use of this in production. Michal W. told me to leave it there just in case, but even if we need something like that, the really important are the methods of source class, and they are all there.
Summing up, we can remove it without regrets ;)